### ISSUE TITLE
Automated build fails

### ISSUE THREAD

**Issue by dimaqq** [jpadilla/pyjwt#483]
```
Run python -m pip install --upgrade pip
Requirement already up-to-date: pip in /opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/site-packages (20.0.2)
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
##[error]Process completed with exit code 1.
```

There's no `requirements.txt` in the repo though, what is it on about?

**Comment by dimaqq** [comment]
Example: https://github.com/jpadilla/pyjwt/pull/482/checks?check_run_id=614074375

**Comment by jpadilla** [comment]
ah yeah, thanks for the reminder. will be fixed in #470
